common4j/src/main/com/microsoft/identity/common/java/util/UrlUtil.java

```diff
@@ -43,15 +43,25 @@
 public class UrlUtil {
     private static final String TAG = UrlUtil.class.getSimpleName();
 
+    public static URL appendPathToURL(@NonNull final URL urlToAppend,
```

Comment:
The method says append path so why is path `Nullable`? It should be NonNull it seems to me

Reply:
It looked like a completely new method being added here